package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "node build/index.js",
-    "dev": "export DEBUG=dev && nodemon --exec babel-node server/index.js",
+    "dev": "export DEBUG=dev && NODE_ENV=production && nodemon --exec babel-node server/index.js",
```

Comment:
I think the former dev script is the once the team uses. The `NODE_ENV=production` should not to be there.